src/Components/CreateImageWizard/utilities/requestMapper.ts

```diff
@@ -558,6 +569,30 @@ const getOpenscap = (state: RootState): OpenScap | undefined => {
   return undefined;
 };
 
+const getUsers = (state: RootState): User[] | undefined => {
+  const users = selectUsers(state);
+
+  if (users.length === 0) {
+    return undefined;
+  }
+
+  return users.map((user) => {
+    const result: User = {
+      name: user.name,
+    };
+    if (user.password !== undefined) {
```

Comment:
Please see what I wrote here:
https://github.com/osbuild/image-builder-frontend/pull/2697#discussion_r1890818172

The password will be stored as a string. We will use `''` as the empty state, not undefined. Therefore, we don't need to check whether or not it is undefined - we need to check whether it is `''` or not.